docker-contributor/Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN apt-get update \
     supervisor apache2-utils lsb-release \
     libcurl4-gnutls-dev libjsoncpp-dev libmagic-dev \
     enscript lpr ca-certificates less vim \
+    php-pear php-dev \
+    && pecl install pcov \
     && rm -rf /var/lib/apt/lists/*
```

Comment:
Don’t we need the COPY for the ini file here as well?

Reply:
Due to the location it should be from this step:
```
# Add PHP configuration
COPY ["php-config", "//etc/php/7.4/fpm/conf.d"]
COPY ["php-config", "//etc/php/7.4/cli/conf.d"]
```

assuming that is copy all the files in that folder,

Reply:
GitHub doesn't show this in the diff, so strange.